core-bundle/src/Command/InstallCommand.php

```diff
@@ -96,4 +104,21 @@ private function addEmptyDir(string $path): void
 
         $this->rows[] = Path::makeRelative($path, $this->projectDir);
     }
+
+    private function getContaoComponentDir(): string|null
+    {
+        $fs = new Filesystem();
+
+        if (!$fs->exists($composerJsonFilePath = Path::join($this->projectDir, 'composer.json'))) {
+            return null;
+        }
+
+        $composerConfig = json_decode(file_get_contents($composerJsonFilePath), true, 512, JSON_THROW_ON_ERROR);
+
+        if (null === ($componentDir = $composerConfig['extra']['contao-component-dir'] ?? null)) {
+            return null;
+        }
+
+        return $componentDir;
```

Comment:
```suggestion
        return $composerConfig['extra']['contao-component-dir'] ?? null;
```

Reply:
Changed in 8c0a770bce935170f3338df31af24fe4ede98956.